### PR TITLE
astromenace: 1.3.2 -> 1.4.1

### PR DESCRIPTION
**Pull request by legendofmiracles**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 150258` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>astromenace</li>
  </ul>
</details>

Builds and runs on my machine (played 3 level or so in tota)

File: pkgs/games/astromenace/default.nix [hunk 1]
```diff
@@ -12,27 +12,29 @@ stdenv.mkDerivation rec {
...
     chmod 755 $out/bin/AstroMenace
   '';
 
-  meta = {
+  meta = with lib; {
     description = "Hardcore 3D space shooter with spaceship upgrade possibilities";
     homepage = "https://www.viewizard.com/";
-    license = lib.licenses.gpl3;
-    platforms = lib.platforms.linux;
+    license = licenses.gpl3;
```
Comment (markuskowa) [inline pkgs/games/astromenace/default.nix:36]: `gpl3` is deprecated. `gpl3Only` or `gpl3Plus`?

Comment (legendofmiracles) [reply pkgs/games/astromenace/default.nix:36]: How can I tell the difference? [Repology can't really decide](https://repology.org/project/astromenace/information#licenses)

Comment (markuskowa) [reply pkgs/games/astromenace/default.nix:36]: It's best to look at the source code (https://github.com/viewizard/astromenace/blob/master/LICENSE.md). The source code itself would then be `gpl3Plus`, as it contains the magic words `either version 3 of the License, or (at your option) any later version.`

**Comment by markuskowa** [comment]
@GrahamcOfBorg eval

**Pull request merged**